web/src/i18n/translations/ja.yml

```diff
@@ -270,18 +270,18 @@ Date picker: 日付ピッカー
 Select from a list of tags: タグのリストから選択する
 Check Box: チェックボックス
 Select from a list of checkboxes: チェックボックスのリストから選択する
-Customize a group of fields: ''
-Delete Group: ''
-Are you sure you want to delete the group: ''
+Customize a group of fields: フィールドのグループをカスタマイズする
+Delete Group: グループ削除
+Are you sure you want to delete the group: 本当にグループを削除しますか？
 Warning: 注意
-This action will permanently delete the selected group and cannot be reversed.: ''
-New Group: ''
-Update Group: ''
-Group name: ''
-Please input the name of the group!: ''
-Group description: ''
-Group key: ''
-Group key must be unique and at least 1 character long. It can only contain letters, numbers, underscores and dashes.: ''
+This action will permanently delete the selected group and cannot be reversed.: この操作は選択したグループを永久に削除し、元に戻すことはできません。
+New Group: 新グループ
+Update Group: 更新グループ
+Group name: グループ名
+Please input the name of the group!: グループ名を入力してください！
+Group description: グループ概要
+Group key: グループキー
+Group key must be unique and at least 1 character long. It can only contain letters, numbers, underscores and dashes.: グループ・キーはユニークで1文字以上でなければなりません。文字、数字、アンダースコア、ダッシュのみを含むことができます。
```

Comment:
```suggestion
Group key must be unique and at least 1 character long. It can only contain letters, numbers, underscores and dashes.: グループキーはユニークで1文字以上でなければなりません。文字、数字、アンダースコア、ダッシュのみを含むことができます。
```

Reply:
Done!